server/fleet/software.go

```diff
@@ -208,6 +208,11 @@ type SoftwareTitle struct {
 	BundleIdentifier *string `json:"bundle_identifier,omitempty" db:"bundle_identifier"`
 }
 
+type UploadSoftwareInstallerResponsePayload struct {
+	TitleID         uint               `json:"software_title_id"`
```

Comment:
Per discussions earlier, since `title_id` exists on SoftwarePackage (if populated), we don't need the title ID at the top level.